pages/email/clients/en.text

```diff
@@ -46,7 +46,7 @@ The [[secure connection -> secure-connections]] may be of type TLS or StartTLS.
 
 *Note:* Don't enable *secure passwords* or *secure authentication*. These are somewhat of a misnomer. These methods of specifying passwords require that the email server keep a cleartext copy of your password. We would consider this a security risk, so we don't enable "secure passwords." Because the connection to riseup.net is encrypted anyway, these are not needed.
 
-*Tor Note:* SMTP port 465 is often blocked by exit nodes, but port 587 is less frequently blocked. If you have a problem sending mail, try 587.
+*Tor Note:* SMTP port 465 is often blocked by exit nodes, but port 587 is less frequently blocked. If you have a problem sending mail, try 587 or configure your client to use Riseup's email hidden service (zsolxunfmbfuq7wf.onion) in place of the regular mail.riseup.net domain. This is better than sending traffic through a Tor exit as it is MITM resistant, but it will generate certificate errors on the client side. 
```

Comment:
`*Tor Note:* SMTP port 465 is often blocked by exit nodes, but port 587 is less frequently blocked. If you have a problem sending mail, try 587 or configure your client to use [[Riseup's email hidden service => tor/hs-addresses-signed.txt]] in place of the regular mail.riseup.net domain. This is better than sending traffic through a Tor exit as it is MITM resistant, but it will generate certificate errors on the client side.`